src/Workspaces/CoreTest/ServicesTest.csproj

```diff
@@ -67,6 +67,8 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="DependentTypeFinderTests.cs" />
+    <Compile Include="EditorConfigStorageLocation\EditorConfigStorageLocationTests.cs" />
```

Comment:
I'm not sure what this is, but I'm guessing you didn't mean to do it?

Reply:
hmm.